visualization/motion_planning_rviz_plugin/src/motion_planning_frame_planning.cpp

```diff
@@ -211,7 +215,14 @@ void MotionPlanningFrame::updateQueryStateHelper(robot_state::RobotState &state,
         else
           if (v == "<same as start>")
           {
-            state = *planning_display_->getQueryStartState();
+            if ( planning_display_->usesCurrentStartState() )
```

Comment:
Why is this extension needed? Probably because the start state wasn't (yet) updated to the current state.
Shouldn't be this update dealt with elsewhere, e.g. directly before planning?